Weather app/ViewModels/ViewModel.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ViewModel.swift
+//  Weather app
+//
+//  Created by Ernest Mihasenko on 27.07.22.
+//
+
+import Foundation
+import CoreLocation
+import Combine
+
+class CurrentWeatherViewModel: NSObject, ObservableObject, CLLocationManagerDelegate {
+    
+    @Published var weatherData: NetworkWeatherData?
```

Comment:
I can't see where you read this object, you only write to it. Maybe you don't need it?